app/controllers/constituencies_controller.rb

```diff
@@ -31,7 +31,10 @@ def show
   def current
     letter_data = Parliament::Request.new.constituencies.current.a_z_letters.get
 
-    @constituencies = Parliament::Request.new.constituencies.current.get.sort_by(:name)
+    data = Parliament::Request.new.constituencies.current.get
+
+    @constituencies = data.filter('http://id.ukpds.org/schema/ConstituencyGroup')
+    @constituencies = @constituencies.sort_by(:name)
```

Comment:
can we combine filtering and sorting in some way? This is 2N (on the filtered list) right?